src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -116,6 +117,56 @@ const SingleContentWrapper = styled.div`
   `}
 `;
 
+const OptionallyRenderedSkipWrapper = ({
+  hasSkipLnk,
+  service,
+  skipLinkTerms,
+  endTextVisuallyHidden,
+  endTextId,
+  skipText,
+  children,
+}) =>
+  hasSkipLnk ? (
+    <SkipLinkWrapper
+      service={service}
+      endTextId={endTextId}
+      text={skipText}
+      endTextVisuallyHidden={endTextVisuallyHidden}
+      terms={skipLinkTerms}
+    >
+      {children}
+    </SkipLinkWrapper>
+  ) : (
+    children
+  );
+
+const optionalSkipLinkProps = {
+  hasSkipLnk: bool,
+  skipLinkTerms: shape({}),
```

Comment:
What does this prop type check for, that it's always an empty object? `{}`

Reply:
it's usually an `enum` of tokenized string used to find and replace certain terms in the skip text, so it could be anything. Since we're using this for recommendations I will just specify what we're passing in instead.